vulcan/lib/server/workflows/scripts/plot_umap.py

```diff
@@ -55,29 +55,32 @@ def get(ids, value):
     color = None
 
 ##### OUTPUT
+plot_args = {
+    'data_frame': scdata.obsm['X_umap'],
+    'x':0, 'y':1,
+    'color_discrete_sequence': colors,
+    'color': color
+    }
+
+# if 'color is dicrete'
```

Comment:
What's a good way to check for discrete vs continuous in python?

Reply:
Do you mean numeric vs. categorical, i.e. number vs. string?

Reply:
Yea, I suppose those are the only options so perhaps a silly question, but I wanted to do it safely without knowing all options.

Reply:
There's also booleans and perhaps other types allowed as a record attribute